README.md

```diff
@@ -93,12 +93,13 @@ integrations in other projects, we'd be happy to have you involved:
 - Join the mailing list: send an email to
   [dev-subscribe@arrow.apache.org][1]. Share your ideas and use cases for the
   project.
-- [Follow our activity on JIRA][3]
+- Follow our activity on [GitHub issues][3] and [GitHub labels][6]
```

Comment:
Is the link to the labels needed? (you also can't follow or watch issues based on the label)

Reply:
No, not needed at all. I just found it nice to see the activity across different components.